packages/widget_toolkit_pin/README.md

```diff
@@ -12,61 +22,206 @@ Step 1: Add the `widget_toolkit_pin`  and `widget_toolkit_biometrics` packages a
 $ flutter pub add widget_toolkit_pin widget_toolkit_biometrics  
 ```
 
-Step 2: Pass the `WidgetToolkitPin` extensions to the `ThemeData` of your app:
+Step 2: Follow the [integration instructions for the local_auth package](https://pub.dev/packages/local_auth) in order for the plugin to work on your desired platforms.
+
+Step 3: Pass the `WidgetToolkitTheme`,`PinCodeTheme` extensions to the `ThemeData` of your app:
 ```dart
 MaterialApp(  
   theme: ThemeData.light().copyWith(  
     colorScheme: ColorScheme.fromSwatch(),
-      WidgetToolkitPin.light,  
+      PinCodeTheme.light,
+      WidgetToolkitTheme.light,  
     ],  
   ),  
   darkTheme: ThemeData.dark().copyWith(  
     colorScheme: ColorScheme.fromSwatch(),  
     extensions: [
-      WidgetToolkitPin.dark,
+      PinCodeTheme.dark,
+      WidgetToolkitTheme.dark,
     ],
   ),  
-); 
+);
+```
+To access the content of the `widget_toolkit_pin`, you should import the files with the following line:
+```dart
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+```
+
+**Note:** The `WidgetToolkitTheme` comes from the **Widget Toolkit** package which already presence in the **Widget Toolkit Pin** package.
+It can be imported with the following line:
+```dart
+import 'package:widget_toolkit/widget_toolkit.dart';
 ```  
 
+Step 3: Create an implementation of `BiometricsLocalDataSource`, for example:
+```dart
+class ProfileLocalDataSource implements BiometricsLocalDataSource { 
+	static const _areBiometricsEnabled = 'areBiometricsEnabled';
+	
+	Future<SharedPreferences> get _storageInstance => SharedPreferences.getInstance();
+	
+	@override  
+	Future<bool> areBiometricsEnabled() async {  
+	  final storage = await _storageInstance;  
+	  return storage.getBool(_areBiometricsEnabled) ?? false;  
+	}
+	
+	@override  
+	Future<void> setBiometricsEnabled(bool enable) async {  
+	  final storage = await _storageInstance;  
+	  await storage.setBool(_areBiometricsEnabled, enable);  
+	}
+}
+```
+
+Step 6: Create an implementation of `PinCodeService`, for example:
+```dart
+class AppPinCodeService implements PinCodeService {
+  AppPinCodeService();
+
+  Future<SharedPreferences> get _storageInstance =>
```

Comment:
Due to security reasons the pin must be stored in the secured storage if that make sense

Reply:
I have changed the storing of the pin code to be in secured storage and before that the pin is encrypted.